### PR TITLE
fix(filestore): add rate limit to event attachments

### PR DESCRIPTION
**Pull request by volokluev**
Currently we do not rate limit how many attachments a project can save, this causes INC-766 and INC-754

The numbers are pulled out of my butt, we'll see if CI passes

### CODE REVIEW

File: src/sentry/event_manager.py [hunk 1]
```diff
@@ -2772,36 +2772,57 @@ def save_attachment(
...
-        size=file.size,
-        sha1=file.sha1,
-        # storage:
-        file_id=file.file_id,
-        blob_path=file.blob_path,
+    is_limited, num_requests, reset_time = ratelimiter.backend.is_limited_with_value(
+        key="event_attachment.save",
+        limit=options.get("sentry.save-event-attachments.project-per-5-minute-limit"),
+        project=project,
+        window=60,
```
Comment (vartec) [inline src/sentry/event_manager.py:2781]: description says 5 minutes, but the window is 60 seconds

**Review by vartec — APPROVED** [review]
other than the window length detail, LGTM

File: src/sentry/event_manager.py (hunk 1 above)
Comment (volokluev) [reply src/sentry/event_manager.py:2781]: Thank you

File: src/sentry/options/defaults.py [hunk 2]
```diff
@@ -2529,6 +2529,14 @@
     "api.organization-activity.brownout-duration", default="PT1M", flags=FLAG_AUTOMATOR_MODIFIABLE
 )
 
+
+register(
+    "sentry.save-event-attachments.project-per-5-minute-limit",
+    type=Int,
+    default=20000,
```
Comment (JoshFerge) [inline src/sentry/options/defaults.py:2536]: i think we should make the default much much higher, or default infinity, and then ratchet it down from there

Comment (volokluev) [reply src/sentry/options/defaults.py]: Done

**Pull request merged**